pkg/apis/migration/v1alpha1/migmigration_types.go

```diff
@@ -17,11 +17,23 @@ limitations under the License.
 package v1alpha1
 
 import (
+	"context"
+	"fmt"
+	"reflect"
+	"time"
+
+	"github.com/fusor/mig-controller/pkg/util"
+	velerov1 "github.com/heptio/velero/pkg/apis/velero/v1"
 	kapi "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
 	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
 )
 
+var clog = logf.Log.WithName("controller")
```

Comment:
Thought you might ask about this... My editor (vscode) automatically refreshes imports according to what you're using in a file whenever you hit `ctrl+s`. Every time I tried to do this with `log` in, it would automatically pull in a prometheus logging package... I probably need to go back and solve that problem with my editor, although have no idea how deep its buried. Changing the name to `clog` (controller log) allowed me to proceed.